doc-files/release-notes-17.md

```diff
@@ -0,0 +1,134 @@
+# Release Notes for JavaFX 17
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+As of JDK 11 the JavaFX modules are delivered separately from the JDK. These release notes cover the standalone JavaFX 17 release. JavaFX 17 requires JDK 11 or later.
+
+## Important Changes
+
+### The src.zip file moved to the top directory of the JavaFX sdk
+
+The `src.zip` file, which is delivered as part of the JavaFX SDK in support of IDEs, has moved from the `lib` directory to the top directory of the JavaFX SDK.
+Application developers may need to adjust their IDE settings so that the IDE can locate the file in its new location.
+
+## List of Enhancement
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8258777](https://bugs.openjdk.java.net/browse/JDK-8258777)|SkinBase: add api to un-/register invalidation-/listChange listeners|controls
+[JDK-8267554](https://bugs.openjdk.java.net/browse/JDK-8267554)|Support loading stylesheets from data-URIs|controls
+[JDK-8223717](https://bugs.openjdk.java.net/browse/JDK-8223717)|javafx printing: Support Specifying Print to File in the API|graphics
+[JDK-8234920](https://bugs.openjdk.java.net/browse/JDK-8234920)|Add SpotLight to the selection of 3D light types|graphics
+[JDK-8259718](https://bugs.openjdk.java.net/browse/JDK-8259718)|Remove the Marlin rasterizer (single-precision)|graphics
+[JDK-8267551](https://bugs.openjdk.java.net/browse/JDK-8267551)|Support loading images from inline data-URIs|graphics
+[JDK-8268120](https://bugs.openjdk.java.net/browse/JDK-8268120)|Allow hardware cursor to be used on Monocle-EGL platforms|graphics
+[JDK-8257895](https://bugs.openjdk.java.net/browse/JDK-8257895)|Allow building of JavaFX media libs for Apple Silicon|media
+[JDK-8258499](https://bugs.openjdk.java.net/browse/JDK-8258499)|JavaFX: Move src.zip out of the lib directory|other
+[JDK-8252935](https://bugs.openjdk.java.net/browse/JDK-8252935)|Add treeShowing listener only when needed|scenegraph
+[JDK-8259680](https://bugs.openjdk.java.net/browse/JDK-8259680)|Need API to query states of CAPS LOCK and NUM LOCK keys|scenegraph
+
+## List of Sub-task
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8092439](https://bugs.openjdk.java.net/browse/JDK-8092439)|[Monocle] Refactor monocle SPI to allow support for multiple screens|graphics
```

Comment:
Since this is just refactoring, I probably wouldn't mention it in the release notes. I added the `noreg-cleanup` label in JBS.

Reply:
The commit does more than refactoring the SPI, so the issue was probably not the best one to link with this PR.

Reply:
I removed the `noreg-cleanup` label.